HousingManagementSystemApi.Tests/ContollersTests/AddressControllerTests.cs

```diff
@@ -9,73 +9,128 @@
 using Moq;
 using Xunit;
 
-namespace HousingManagementSystemApi.Tests.ContollersTests
+namespace HousingManagementSystemApi.Tests.ControllersTests
```

Comment:
I'm getting here `Namespace does not correspond to file location, should be: 'HousingManagementSystemApi.Tests.ContollersTests'` from Rider.